From 1660d8c4986d6c6344a62abf7b3656b31110fdb9 Mon Sep 17 00:00:00 2001
From: Xie Chao <chao.xie@intel.com>
Date: Fri, 9 Aug 2019 13:38:24 +0800
Subject: [PATCH 5/8] remove check for some fs mount

AIC does not need mount sysfs proc and selinuxfs

Change-Id: I0dec10425aae327b3bbd51428b449d9818e9393d
Tracked-On:
Signed-off-by: Xie Chao <chao.xie@intel.com>
---
 init/first_stage_init.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/init/first_stage_init.cpp b/init/first_stage_init.cpp
index 0e68408d1..79fd9d148 100644
--- a/init/first_stage_init.cpp
+++ b/init/first_stage_init.cpp
@@ -122,7 +122,7 @@ int FirstStageMain(int argc, char** argv) {
     CHECKCALL(mkdir("/dev/socket", 0755));
     CHECKCALL(mount("devpts", "/dev/pts", "devpts", 0, NULL));
 #define MAKE_STR(x) __STRING(x)
-    CHECKCALL(mount("proc", "/proc", "proc", 0, "hidepid=2,gid=" MAKE_STR(AID_READPROC)));
+    mount("proc", "/proc", "proc", 0, "hidepid=2,gid=" MAKE_STR(AID_READPROC));
 #undef MAKE_STR
 #ifdef _ANDROID_AS_GUEST_
     CHECKCALL(mount("/cmdline", "/proc/cmdline", NULL, MS_BIND|MS_RDONLY, NULL));
@@ -131,8 +131,8 @@ int FirstStageMain(int argc, char** argv) {
     CHECKCALL(chmod("/proc/cmdline", 0440));
     gid_t groups[] = {AID_READPROC};
     CHECKCALL(setgroups(arraysize(groups), groups));
-    CHECKCALL(mount("sysfs", "/sys", "sysfs", 0, NULL));
-    CHECKCALL(mount("selinuxfs", "/sys/fs/selinux", "selinuxfs", 0, NULL));
+    mount("sysfs", "/sys", "sysfs", 0, NULL);
+    mount("selinuxfs", "/sys/fs/selinux", "selinuxfs", 0, NULL);
 
     CHECKCALL(mknod("/dev/kmsg", S_IFCHR | 0600, makedev(1, 11)));
 
-- 
2.21.0

